backend/src/main/java/org/launchcode/fancyrats/controllers/TestController.java

```diff
@@ -27,6 +27,12 @@ public String moderatorAccess() {
         return "Moderator Board.";
     }
 
+    @GetMapping("/jobs")
+    @PreAuthorize("hasRole('MODERATOR')")
+    public String jobAccess() {
+        return "Job created";
+    }
```

Comment:
What page is this route for?

Reply:
It's for the jobs page. Should have @GetMapping("/jobs")

Reply:
I already have an endpoint for this:
https://github.com/TA-Grant1101/fancy-rats/blob/769ef3b74116fb1d647ab2fd00a78b7705d6d766/backend/src/main/java/org/launchcode/fancyrats/controllers/JobsController.java#L39
I don't think we want to keep separate controllers for different request types instead of routing them all through the test controller